gems/smithy-cbor/lib/smithy-cbor/codec.rb

```diff
@@ -12,22 +12,27 @@ module CBOR
     #   * Update (de)serializing document types
     #   * Allow user to pass in their preferred type to deserialize
     #     If it fails, resort to deserializing type on the shape.
-    module Codec
+    class Codec
```

Comment:
Since you are already touching this file - can you remove "support handling of typed documents" and "Update (de)serializing document types" TODOs - RPCv2CBOR does not support Document shapes when I was looking into document types. Thanks!